lib/actions-oclif/apps.ts

```diff
@@ -46,21 +47,25 @@ export default class AppsCmd extends Command {
 
 	public static flags: flags.Input<FlagsDef> = {
 		help: cf.help,
+		verbose: flags.boolean({
+			char: 'v',
+			description: 'add extra columns in the tabular output (SLUG)',
```

Comment:
If we just added the `slug` column in the output without a `--verbose` option, this would technically be a breaking change that required a major version release (because users parse the CLI output in shell scripts with `awk`, `sed`, etc). But we can add the SLUG column by default in the next major version release: see #1770